docs/.doxygen/Doxyfile

```diff
@@ -44,7 +44,7 @@ PROJECT_NUMBER         = v4.0.5.0
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = "prototype interfaces compatible with ROCm platform and HiP"
+PROJECT_BRIEF          = "prototype interfaces compatible with ROCm software and HiP"
```

Comment:
I think "HiP" should be "HIP", might as well update here.

Reply:
agreed!

Reply:
Thanks, looks good. Feel free to squash and merge.